src/DssProxy.sol

```diff
@@ -24,11 +24,13 @@ contract DssProxy {
     address public owner;
     address public authority;
 
-    event SetOwner(address indexed owner);
-    event SetAuthority(address indexed authority);
+    event SetOwner(address indexed owner_);
+    event SetAuthority(address indexed authority_);
+    event Execute(address indexed target_, bytes data_);
```

Comment:
Small note that the gas cost for logging the entire data is 8 per byte (I know we had it before as well):
[https://github.com/wolflo/evm-opcodes/blob/main/gas.md?plain=1#L229](https://github.com/wolflo/evm-opcodes/blob/main/gas.md?plain=1#L229)

Reply:
Yes, I was wondering about it, we might just want to delegate the logging to the proxy action whenever is needed.

Reply:
Right, maybe we should just ask in the integration channel if anyone is relying on this event and otherwise remove it.